book/scheduling.md

```diff
@@ -450,36 +458,36 @@ to running JavaScript.
 
 ::: {.further}
 
-While it looks simple and maybe even obvious in retrospect, the `XMLHttpRequest`
-API played a key role in the web's evolution beyond the "90s web" that relied on
-loading new pages whenever anyone clicked a link or submitted a form. With the
-async version of this API, web pages were able to act a whole lot more like
-an *application* than a page of information. This ushered in a new generation
-of web sites that used this technique; GMail is one famous early example that
-dates from April 2004, [soon after][xhr-history] enough browsers finished adding
-support for the API.
-
+`XMLHttpRequest` played a key role in helping the web evolve. In the
+90s, clicking on a link or submitting a form required loading a new
+pages. With `XMLHttpRequest` web pages were able to act a whole lot
+more like a dynamic application, with GMail one famous early
+example.[^when-gmail] Nowadays, a web application that uses DOM
+mutations instead of page loads to update its state is called a
+[single-page app][spa]. SPAs enabled more interactive and complex web
+apps, which in turn made browser speed and responsiveness more
+important.
+
+[^when-gmail]: GMail dates from April 2004, [soon after][xhr-history]
+enough browsers finished adding support for the API. The first
+application to use `XMLHttpRequest` was [Outlook Web Access][outlook],
+in 1999, but it took a while for the API to make it into other
+browsers.
+
+[outlook]: https://en.wikipedia.org/wiki/Outlook_on_the_web
 [xhr-history]: https://en.wikipedia.org/wiki/XMLHttpRequest#History
-
-These new applications used an approach that is now called a
-[single-page app][spa], or SPA, as opposed to the earlier multi-page app, or
-MPA. An SPA uses DOM mutations instead of page loads to update to the latest
-app state. The emergence of SPAs enabled more interactive and complex
-web apps, which in turn greatly increased the need for browser rendering to be
-faster and more efficiently scheduled.
-
 [spa]: https://en.wikipedia.org/wiki/Single-page_application
 
 :::
 
 The cadence of rendering
 ========================
 
-There's more to tasks than just implementing some JavaScript APIs. Once
-something is a `Task`, the task runner controls when it runs: perhaps now,
-later, at most once a second, at different rates for active and inactive pages,
-or according to its priority. A browser could even have multiple task runners,
-optimized for different use cases.
+There's more to tasks than just implementing some JavaScript APIs.
+Once something is a `Task`, the task runner controls when it runs:
```

Comment:
You literally undid the edit I added yesterday. I added "perhaps" because it was unclear whether the use of "now" is in the "now, let me tell you something..." meaning vs "right this moment"

Reply:
Oy, my bad. I'll add back "perhaps" and then make it parallel in the other options.